README.md

```diff
@@ -33,49 +39,72 @@ This repository is organized as follows:
+- Since there is currently no code that supports API versioning, this repo will remain unversioned. Each swagger definition can define it's own api version, and consumers may test API changes on a branch, but until there is support for versioned APIs in code, we won't tag this repo.
 
-#### Other swagger definitions?
- - The goal of this repo is to not only provide a central, trusted location for consumers of our external APIs, but also to provide a re-usable means for service maintainers to easily maintain their service API definitions. **As repository adoption increases, we expect to remove duplicate definitions from our organization's codebase where appropriate, rather than provide API definitions in multiple locations.**
+### Gateway?
 
+- Gateway refers to the entry point into our platform (whether it be a reverse proxy or some other router).
 
-## How to use:
+### Flow of an external request?
 
-#### Prologue:
+- [client] -> [gateway] -> [service]
+  - An external `client` makes a request for `/api/v2/thing`.
+  - `gateway` finds where to handle requests for `/thing` and forwards the request to that service.
+  - `service` then handles that request and responds to the client.
+
+### Flow of an internal request?
+
+- [service] -> [service]
+  - A `service` running inside the platform makes a request to another platform `service` at `/thing`.
+  - `service` then handles that request and responds to the client (requesting `service`).
+
+### Location?
+
+- Logic states that the further the API spec is from the implementing code, the greater the potential drift between the two. There are several reasons the service specific ("internal") definitions are in this repo and not living next to the code they define. Primarily it comes down to standardization. By standardizing on where to locate all swagger definitions, consumers aren't left wondering where they need to look for the current API contract of a service. Another equally important reason is more technical. While it is possible to reference swagger components via URL (as opposed to local relative file), the fact that many of the services are closed source re-introduce the issue of consuming private swagger. Third slightly relates to the last reason, as it simplifies the integration of the service into the "external" API (no reference path update required).
```

Comment:
```suggestion
- Logic states that the further the API spec is from the implemented code, the greater the potential drift between the two. There are several reasons the service specific ("internal") definitions are in this repo and not living next to the code they define. Primarily it comes down to standardization. By standardizing on where to locate all swagger definitions, consumers aren't left wondering where they need to look for the current API contract of a service. Another equally important reason is more technical. While it is possible to reference swagger components via URL (as opposed to local relative file), the fact that many of the services are closed source re-introduce the issue of consuming private swagger. Third slightly relates to the last reason, as it simplifies the integration of the service into the "external" API (no reference path update required).
```

Reply:
@kelseiv @sunbryely-influxdata Please look at ./docs/pathTemplate.yml again and let me know what you think. I still have some cleanup to do, but I don't expect to move or add much else for now.

I also still need to add @kelseiv's suggestions in the README.